lib/Dialect/ESI/AppID.cpp

```diff
@@ -58,12 +53,18 @@ class AppIDIndex::ModuleAppIDs {
   // Returns a range iterator to the AppID components exposed by this module.
   auto getAppIDs() const { return llvm::make_first_range(childAppIDPaths); }
 
+  // Get a read-only reference to the index.
+  const auto &getChildren() const { return childAppIDPathsOrdered; }
```

Comment:
Why not juts `ArrayRef`?

Reply:
Originally the return type was more complex.